src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -30,6 +31,14 @@ private static bool IsNet70OrGreater()
 
         private static string IncludeExpectedOutput(string expectedOutput) => IsNet70OrGreater() ? expectedOutput : null;
 
+        private static IEnumerable<MetadataReference> CopyWithoutSharingCachedSymbols(IEnumerable<MetadataReference> references)
```

Comment:
Why is this necessary? It wasn't necessary for any previous feature that set override flags.

Reply:
This is used in two tests that need more than mscorlib. For example, one needs `dynamic` machinery.
Unless those previous features tests such combinations, they would not run into this situation.
I can't say that I understand `CopyWithoutSharingCachedSymbols` well enough to say there isn't a better way to do this. I'm happy to open a tracking issue to re-discuss when Aleksey is back.

Reply:
Filed [https://github.com/dotnet/roslyn/issues/63222](https://github.com/dotnet/roslyn/issues/63222) to review usages of `CopyWithoutSharingCachedSymbols()`